examples/ffva/src/main.c

```diff
@@ -312,7 +312,7 @@ void vApplicationMallocFailedHook(void)
 static void mem_analysis(void)
 {
 	for (;;) {
-		rtos_printf("Tile[%d]:\n\tMinimum heap free: %d\n\tCurrent heap free: %d\n", THIS_XCORE_TILE, xPortGetMinimumEverFreeHeapSize(), xPortGetFreeHeapSize());
+		//rtos_printf("Tile[%d]:\n\tMinimum heap free: %d\n\tCurrent heap free: %d\n", THIS_XCORE_TILE, xPortGetMinimumEverFreeHeapSize(), xPortGetFreeHeapSize());
```

Comment:
Would rather delete commented lines, or declare a debug unit for debug_printf that disables these printouts... can we do that for rtos_printf? unsure

Reply:
I meant to uncomment these before merging and forgot. Sorry, will do it now